docs/src/emulation.md

```diff
@@ -792,5 +792,5 @@ context = browser.new_context(
 ```
 
 ```csharp
-var context = await browser.NewContextAsync(new() { JavaScriptEnabled = true });
+var context = await browser.NewContextAsync(new() { JavaScriptEnabled = false });
```

Comment:
Let's fix the snippet in line 784 and 790 too? It should be `java_script_enabled`.

Reply:
done :)